docs/overview.md

```
@@ -0,0 +1,332 @@
+いどばたビジョンは複数のAIモデルを用途に応じて使い分けています。すべてのLLM（大規模言語モデル）呼び出しはOpenRouter API経由で行われます。
+
+| モデル | 用途 |
+|--------|------|
+| `google/gemini-2.0-flash-001` | チャット応答・課題/解決策の抽出・論点リンキング（日常的な処理に使用） |
+| `google/gemini-2.5-pro-preview-03-25` | 重要論点生成・政策ドラフト・ダイジェスト・レポート生成・論点分析（高精度が必要な処理に使用） |
+| `anthropic/claude-3.7-sonnet` | ビジュアルレポート（HTML形式のイラスト要約生成） |
+| `text-embedding-3-small`（OpenAI） | テキストの埋め込みベクトル生成（python-serviceが使用） |
+
+### AI処理パイプラインの全体フロー
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)